.github/workflows/finale.js

```diff
@@ -25,7 +25,7 @@ async function main() {
   // 20 minutes * 60 seconds/min / 5 seconds per loop.
   const iterations = 20 * 60 / 5
   for (let i=0; i<iterations; i++) {
-    const url = `https://api.github.com/repos/${repo}/actions/runs/${runId}/jobs`;
+    const url = `https://api.github.com/repos/${repo}/actions/runs/${runId}/jobs?per_page=100`;
```

Comment:
This seems unrelated - it may or may not be a part of this PR

Reply:
I needed that to make the CI pass. This PR needs to make sure more than 30 CI jobs succeed and the default page size was 30.